factcast-core/src/main/java/org/factcast/core/subscription/SubscriptionRequestTO.java

```diff
@@ -42,7 +42,9 @@
 
   private static final String PID = ManagementFactory.getRuntimeMXBean().getName();
 
-  @JsonProperty long maxBatchDelayInMs = 0;
+  @JsonProperty long maxBatchDelayInMs = 20;
```

Comment:
10 or 20? 🤔

Reply:
tought we can go for 20 as a default (concerned about load on database, rather than 10ms more latency)
what do you think?